src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
```

Comment:
better to use `this` instead of the `calculator` here, it will improve the reusability